tutorials/server/get-started-with-corda/get-started-with-corda.md

```diff
@@ -0,0 +1,196 @@
+---
+id: get-started-with-Corda
+summary: Learn how to get started with Corda, an open source permissioned blockchain platform.
+categories: server
+tags: corda, blockchain, opensource
+difficulty: 3
+status: draft
+image: https://www.corda.net/wp-content/uploads/2017/01/16-10-31_R3_Corda_Master-Logo-04-1-e1516308027926-1024x455.png
+feedback_url: https://github.com/canonical-websites/tutorials.ubuntu.com/issues
+published: 2018-06-19
+author: Jonathan Sphar <Jonathan.Sphar@r3.com>
+
+---
+
+# Get started with Corda
+
+## Overview
+Duration: 1:00
+
+Corda is a blockchain platform built for business. Corda removes costly friction in business transactions by enabling businesses to transact directly. Using smart contract and blockchain technology, Corda allows existing business networks to reduce transaction and record-keeping costs and to streamline business operations. 
+
```

Comment:
What about the Corda logo here? (see https://tutorials.ubuntu.com/tutorial/continuous-snap-delivery-from-travis-ci#0 for example)